tools/travis_script.sh

```diff
@@ -9,7 +9,10 @@ section_end "Test.with.min.requirements"
 
 section "Build.docs"
 if [[ ($PY != 2.6) && ($PY != 3.2) ]]; then
-    make html
+    SPHINXBUILD="python $(which sphinx-build)"
```

Comment:
abd0f48 used backticks and got a similar result as dc394ab. Am I missing something obvious?